llvm/test/CodeGen/AArch64/ptrauth-tiny-model-pic.ll

```diff
@@ -0,0 +1,182 @@
+; RUN: llc -verify-machineinstrs -mtriple=aarch64 -mattr=+pauth -mattr=+fpac -code-model=tiny \
```

Comment:
Perhaps combine pic and static .ll tests and just use update_llc_test_checks.py

Reply:
> Perhaps combine pic and static .ll tests

While we can do this, I'm not sure if it would give that much value: RUN lines are different (`-relocation-model=pic` is used in ptrauth-tiny-model-pic.ll) so we'll have x2 RUN lines when combined to one file which might be overwhelming IMHO, and also we'll have to change check prefixes so they don't overlap

> and just use update_llc_test_checks.py

I was unable to make update_llc_test_checks.py produce nice output which shows that there are common CHECK lines and different TRAP and NOTRAP lines for `-mattr=+fpac` missing and present correspondingly. We can have duplicate check lines for both cases, but it consumes more space and doesn't explicitly show that the only difference between fpac and no-fpac case is the absence/presence of trap sequence.

Please let me know if I miss smth and update_llc_test_checks.py is capable of generating such outputs (and I'm just missing smth)